cherrypy/_cperror.py

```diff
@@ -214,18 +214,7 @@ def __init__(self, urls, status=None, encoding=None):
         if isinstance(urls, text_or_bytes):
```

Comment:
You should keep track of patches you apply to CherryPy for two reasons.
- You'll need to reapply for the next CP release.
- Some packagers assume they can use the official distribution of CP. If not they like to know what patches are needed.